pkgs/by-name/co/cosmic-launcher/package.nix

```diff
@@ -1,70 +1,62 @@
 {
   lib,
   stdenv,
-  stdenvAdapters,
   fetchFromGitHub,
   rustPlatform,
   just,
   libcosmicAppHook,
   nix-update-script,
-
-  withMoldLinker ? stdenv.targetPlatform.isLinux,
 }:
 
-rustPlatform.buildRustPackage.override
-  { stdenv = if withMoldLinker then stdenvAdapters.useMoldLinker stdenv else stdenv; }
-  (finalAttrs: {
-    pname = "cosmic-launcher";
-    version = "1.0.0-alpha.6";
-
-    src = fetchFromGitHub {
-      owner = "pop-os";
-      repo = "cosmic-launcher";
-      tag = "epoch-${finalAttrs.version}";
-      hash = "sha256-BtYnL+qkM/aw+Air5yOKH098V+TQByM5mh1DX7v+v+s=";
-    };
-
-    useFetchCargoVendor = true;
-    cargoHash = "sha256-g7Qr3C8jQg65KehXAhftdXCpEukag0w12ClvZFkxfqs=";
-
-    nativeBuildInputs = [
-      just
-      libcosmicAppHook
+rustPlatform.buildRustPackage (finalAttrs: {
+  pname = "cosmic-launcher";
+  version = "1.0.0-alpha.6";
+
+  src = fetchFromGitHub {
+    owner = "pop-os";
+    repo = "cosmic-launcher";
+    tag = "epoch-${finalAttrs.version}";
+    hash = "sha256-BtYnL+qkM/aw+Air5yOKH098V+TQByM5mh1DX7v+v+s=";
+  };
+
+  useFetchCargoVendor = true;
+  cargoHash = "sha256-g7Qr3C8jQg65KehXAhftdXCpEukag0w12ClvZFkxfqs=";
+
+  nativeBuildInputs = [
+    just
+    libcosmicAppHook
+  ];
+
+  dontUseJustBuild = true;
+  dontUseJustCheck = true;
+
+  justFlags = [
+    "--set"
+    "prefix"
+    (placeholder "out")
+    "--set"
+    "bin-src"
+    "target/${stdenv.hostPlatform.rust.cargoShortTarget}/release/cosmic-launcher"
+  ];
+
+  passthru.updateScript = nix-update-script {
```

Comment:
```suggestion
  ];
  
  env."CARGO_TARGET_${stdenv.hostPlatform.rust.cargoEnvVarTarget}_RUSTFLAGS" = "--cfg tokio_unstable";

  passthru.updateScript = nix-update-script {
```

Reply:
Oh yes sorry! Although do we need it when it builds without it?

Reply:
Put it back in this commit anyway because of course it's nothing to do with mold.

Reply:
> Oh yes sorry! Although do we need it when it builds without it?

I don't think so, I think `.cargo/config.toml` automatically adds this build flag. Maybe can be removed in a separate commit

Reply:
I'll leave it for now then.